src/qt/createwalletdialog.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#if defined(HAVE_CONFIG_H)
+#include <config/bitcoin-config.h>
+#endif
+
+#include <qt/askpassphrasedialog.h>
+#include <qt/bitcoingui.h>
+#include <qt/createwalletdialog.h>
+#include <qt/forms/ui_createwalletdialog.h>
+#include <wallet/wallet.h>
+
+#include <QMessageBox>
+#include <QProgressDialog>
+#include <QString>
+
+CreateWalletDialog::CreateWalletDialog(BitcoinGUI* parent) :
+    QDialog(parent),
+    ui(new Ui::CreateWalletDialog),
+    m_parent(parent)
+{
+    ui->setupUi(this);
+}
```

Comment:
You should use `setWindowTitle()` here so the dialog isn't just titled 'Dialog'.

Reply:
Done